jenkins/job-configs/kubernetes-jenkins-pull/global.yaml

```diff
@@ -1,13 +1,28 @@
+- builder:
+    name: ensure-upload-to-gcs-script
+    builders:
+      - shell: |
+          _script_tmp_path='./_tmp/upload-to-gcs.sh'
+          _script_repo_path='hack/jenkins/upload-to-gcs.sh'
+          mkdir -p ./_tmp
+          if [[ "${{ghprbTargetBranch:-}}" == 'master' && -x "./kubernetes/${{_script_repo_path}}" ]]; then
```

Comment:
for anything that's not the master branch, we want to use the latest version on master.

this is also a fail-safe in case the file is missing or deleted somehow. if it's gone we get no build log, so no way of knowing what happened.